### PR TITLE
Convert server/startup to JS

### PR DESCRIPTION
**Pull request by rodrigok**
@RocketChat/core

### CODE REVIEW

File: server/startup/i18n-validation.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+// 	if (errors.length > 0) {
+// 		console.log('+'.red + s.rpad('', len - 28, '-').red + '+'.red);
+
+// 		for (const error of errors) {
+// 			console.log('|'.red, s.rpad('' + error, len).red, '|'.red);
+// 		}
+
+// 		return console.log('+'.red + s.rpad('', len - 28, '-').red + '+'.red);
+// 	}
+// });
```
Comment (marceloschmidt) [inline server/startup/i18n-validation.js]: Any reason why this file is all commented-out instead of deleted?

File: server/startup/initialData.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
+Meteor.startup(function() {
+	Meteor.defer(function() {
+		if (RocketChat.models.Rooms.findOneById('GENERAL') == null) {
```
Comment (marceloschmidt) [inline server/startup/initialData.js]: Did you intentionally use `==` instead of `===`?

File: server/startup/i18n-validation.js (hunk 1 above)
Comment (rodrigok) [inline server/startup/i18n-validation.js]: That was a code to check missing translations. Maybe we will need it in the future. What do you think?

File: server/startup/initialData.js (hunk 2 above)
Comment (rodrigok) [inline server/startup/initialData.js]: Nops, fixed

File: server/startup/i18n-validation.js (hunk 1 above)
Comment (marceloschmidt) [inline server/startup/i18n-validation.js]: Could we maybe add that as a function that we may run whenever we want?

**Pull request merged**